.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: 'Push CI'
 
 on:
   push:
+    # Avoid tags
+    branches:
+      - "*"
```

Comment:
Testy sa sputaju vramci `CLI/templates/buffer` release, nie je potrebne ich spustat este samostatne, ked to otagujeme.

https://stackoverflow.com/questions/70743715/how-do-i-configure-a-github-actions-workflow-so-it-does-not-run-on-a-tag-push